terra-workspace-manager/api-services.tf

```diff
@@ -1,5 +1,5 @@
 module "enable-services" {
-  source      = "github.com/broadinstitute/terraform-shared.git//terraform-modules/api-services?ref=services-0.3.0-tf-0.12"
+  source      = "github.com/broadinstitute/terraform-shared.git//terraform-modules/api-services?ref=jaycarlton/PF-810"
```

Comment:
I don't think you want to change this even in this draft unless you're planning on modifying the github.com/broadinstitute/terraform-modules repo, which shouldn't be necessary.

Reply:
You're right; that branch doesn't even exist (hopefully) in that other repo.